UIKit/Tasks/RegistrationViewController.swift

```diff
@@ -0,0 +1,53 @@
+// RegistrationViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Класс Вью Контроллер скрывает пароль
+final class RegistrationViewController: UIViewController {
+    // MARK: - IBOutlets
+
+    @IBOutlet private var passwordField: UITextField!
+    @IBOutlet var loginField: UITextField!
+    @IBOutlet private var buttonTappedotlet: UIButton!
+    @IBOutlet var enterButton: UIButton!
+
+    // MARK: - Life Cycle
+
+    override func viewDidLoad() {
+        passwordField.delegate = self
+        loginField.delegate = self
+        enterButton.isEnabled = false
+    }
+
+    // MARK: - Private Methods
+
+    @IBAction private func hidingButton(_ sender: Any) {
+        if buttonTappedotlet.currentImage == UIImage(named: "VectorOpen") {
```

Comment:
```suggestion
        if buttonTappedOutlet.currentImage == UIImage(named: "vectorOpen") {
```
У тебя может shift залипает, так часто с маленькой буквы?)
Имена картинок и цветов с маленькой буквы